hack/update-csv.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import yaml
+
+_VERSIONED_NAME = 'ssp-operator.vPLACEHOLDER_CSV_VERSION'
+_NAMESPACE = 'kubevirt'
+_SPEC = {
+    'version': 'vPLACEHOLDER_CSV_VERSION'
```

Comment:
I think the version should be `PLACEHOLDER_CSV_VERSION`, without the 'v' prefix.

Reply:
This depends on HCO if it sends v prefix or not

Reply:
The `spec.version` field should contain just the semantic version, without the 'v' prefix. If HCO adds it, then it is probably a bug. Look at the generated `bundle/manifests/ssp-operator.clusterserviceversion.yaml`.

Reply:
Ok. Ignore my previous comment. I thought that the `spec.version` must not start with 'v', because it does not in the generated csv in `bundle/manifests/ssp-operator.clusterserviceversion.yaml`. But I found that the API code that parses semantic version supports both with and without the 'v' prefix.